resen/resencmd.py

```diff
@@ -190,29 +174,52 @@ def do_status(self,args):
 
     def do_start_jupyter(self,args):
         """Usage:
->>> start_jupyter bucket_name local_port bucket_port\t: Start a jupyter notebook server on port bucket_port available at local_port.
->>> start_jupyter bucket_name local_port bucket_port --lab\t: Start a jupyter lab server on port bucket_port available at local_port.
+>>> start_jupyter bucket_name
```

Comment:
This doc string should conform to the format of similar commands.
start_jupyter bucket_name : Start jupyter on bucket bucket_name

Reply:
done